dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackedEntityAttributeControllerTest.java

```diff
@@ -166,6 +167,102 @@ void getIndexableOnlyAttributes()
 
     }
 
+    @Test
+    void getIndexableOnlyAttributesWithDateAndTextValueType()
+    {
+
+        teaA.setValueType( ValueType.DATE );
+        manager.update( teaA );
+        teaB.setValueType( ValueType.TEXT );
+        manager.update( teaB );
+        JsonObject json = GET( "/trackedEntityAttributes?indexableOnly=true" ).content( HttpStatus.OK );
+        assertFalse( json.isEmpty() );
+
+        Set<String> expected = Set.of( "AttributeB" );
```

Comment:
I think you can inline expected. Saves a line and is still as (if not more readable).